serde-generate/src/java.rs

```diff
@@ -221,9 +221,9 @@ where
                 self.quote_types(formats)
             ),
             TupleArray { content, size } => format!(
-                "{} @com.novi.serde.ArrayLen(length={}) []",
-                self.quote_type(content),
-                size
+                "java.util./*@com.novi.serde.ArrayLen(length={})*/List<{}>",
```

Comment:
LoL, actually works?!

Reply:
I got the idea from this thread: https://stackoverflow.com/questions/21385938/java-scoping-construct-cannot-be-annotated-with-type-use

but now I realize we are already using Java >=8 so let me fix that.